fdbclient/Tuple.cpp

```diff
@@ -59,7 +61,7 @@ static void adjustFloatingPoint(uint8_t* bytes, size_t size, bool encode) {
 	}
 }
 
-Tuple::Tuple(StringRef const& str, bool exclude_incomplete) {
+Tuple::Tuple(StringRef const& str, bool exclude_incomplete, bool include_user_type) {
```

Comment:
Do we need to restrict access to the user type like this? In general we've left it up to the user of Tuple to determine that the types present match expectations.

Reply:
I think there's just a little hesitation making this sweeping change right up to the release. Doing it this way lets us only use it where we intend to and then in a future release we can remove the guard.